bip-0352.mediawiki

```diff
@@ -5,7 +5,7 @@
   Author: josibake <josibake@protonmail.com>
           Ruben Somsen <rsomsen@gmail.com>
   Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-0352
-  Status: Proposed
+  Status: Final
```

Comment:
If this updated moves forward, the README would need to be updated as well (in the same commit, and in this pull, not a new one)